libs/designsystem/src/lib/components/icon/icon.component.ts

```diff
@@ -50,6 +50,18 @@ export class IconComponent implements OnChanges {
     }
   }
 
+  private showWarning(): string {
+    let warning: string = '';
+
+    if (this.customName) {
+      warning = `Custom icon with name "${this.customName}" was not found. Do you have a typo in 'customName' or forgot to configure the custom icon through the 'IconRegistryService'?`;
+    } else {
+      warning = `Built-in icon with name "${this.name}" was not found. Do you have a typo in 'name' or did you mean to use a custom icon? If so, please use: <kirby-icon customName="${this.name}"></kirby-icon>`;
+    }
+
+    return warning;
+  }
+
```

Comment:
Consider readability:
```suggestion
    if (this.customName) {
      return `Custom icon with name "${this.customName}" was not found. Do you have a typo in 'customName' or forgot to configure the custom icon through the 'IconRegistryService'?`;
    }
      return `Built-in icon with name "${this.name}" was not found. Do you have a typo in 'name' or did you mean to use a custom icon? If so, please use: <kirby-icon customName="${this.name}"></kirby-icon>`;
  }

```

Reply:
Added "line-break" to template string for readability